docs/reference/redirects.asciidoc

```diff
@@ -388,19 +380,19 @@ See <<commands>>.
 === Calendar resources
 
 See <<ml-get-calendar>> and
-{ml-docs}/ml-ad-run-jobs.html#ml-ad-calendars[Calendars and scheduled events].
+{ml-docs}/ml-ad-finding-anomalies.html[Calendars and scheduled events].
```

Comment:
> This change should be reverted, since the original is correct: https://www.elastic.co/guide/en/machine-learning/8.0/ml-ad-run-jobs.html#ml-ad-calendars